meta-refkit-computervision/recipes-devtools/opengl/python-pyopengl_3.0.1.bb

```diff
@@ -0,0 +1,17 @@
+DESCRIPTION = "Standard OpenGL bindings for Python"
+SECTION = "devel/python"
+PRIORITY = "optional"
+DEPENDS = "python-numeric"
+LICENSE = "BSD"
+SRCNAME = "PyOpenGL"
+LIC_FILES_CHKSUM = "file://license.txt;md5=35e5f53336fec095a8a3f229568e151b"
+
+SRC_URI = "\
+  http://pypi.python.org/packages/source/P/${SRCNAME}/${SRCNAME}-${PV}.tar.gz \
+"
+S = "${WORKDIR}/${SRCNAME}-${PV}"
+
+inherit distutils
```

Comment:
You ought to use Python 3. We aim to keep the various components scriptable with a single scripting language (Python 3) to make the interoperability easier.